docs-content/release-notes.html.md.erb

```diff
@@ -5,6 +5,48 @@ owner: Partners
 
 These are release notes for CyberArk Conjur Service Broker for VMware Tanzu
 
+## <a id="ver"></a> v1.3.0
+
+**Release Date:** August 9, 2021
+
+- CyberArk Conjur Service Broker - [v1.2.1](https://github.com/cyberark/conjur-service-broker/releases/v1.2.1)
+- CyberArk Conjur Buildpack - [v2.2.1](https://github.com/cyberark/cloudfoundry-conjur-buildpack/releases/v2.2.1)
+
+Features in this release:
+
+* Removed support for Conjur Enterprise v4
+  - We recommend migrating to Conjur Enterprise v11+ or Conjur OSS v1+.
+  - [cyberark/conjur-service-broker#203](https://github.com/cyberark/conjur-service-broker/issues/203)
+
+* Add organization_name and space_name annotations
+  - Service Broker API spec 2.15 and above provide `organization_name` and
+    `space_name`. If these are available, they are added as annotations on the
+    organization and space policies that are created in Conjur. [cyberark/conjur-service-broker#238](https://github.com/cyberark/conjur-service-broker/issues/238)
+  - Requires Conjur Open Source v1.3.7+ or Conjur Enterprise (formerly Dynamic Access Provider) v11.3.0+
+
+* Support for using Summon environments in secrets.yml file
+  - secrets.yml files can be split into sections by environment and secrets to
+    load at runtime can be specified using new SECRETS_YAML_ENVIRONMENT
+    variable. [cyberark/cloudfoundry-conjur-buildpack#44](https://github.com/cyberark/cloudfoundry-conjur-buildpack/issues/44)
+
+* Fixed error messages with invalid line numbers
+  - Caused by Cloudfoundry Buildpack encountering errors while parsing environment
+    variable settings after retrieving secrets from Conjur. [cyberark/cloudfoundry-conjur-buildpack#120](https://github.com/cyberark/cloudfoundry-conjur-buildpack/issues/120)
+
```

Comment:
Should there be a note about the addition of a `manifest.yml` "allowing for the usage of buildpack-packager and other native CloudFoundry features"? From the **Added** section of [2.1.6 changelog](https://github.com/cyberark/cloudfoundry-conjur-buildpack/compare/v2.1.5...v2.1.6)